paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -693,9 +699,12 @@ internal abstract class BaseSheetViewModel(
 
         return paymentMethodMetadata.value?.formElementsForCode(
             code = code,
-            context = getApplication(),
-            paymentMethodCreateParams = currentSelection?.paymentMethodCreateParams,
-            paymentMethodExtraParams = currentSelection?.paymentMethodExtraParams,
+            uiDefinitionFactoryArgumentsFactory = UiDefinitionFactory.Arguments.Factory.Default(
```

Comment:
I'm not in love with this name... Open to ideas.

Reply:
hmm yeah I get that... it's pretty descriptive though which I think is good